src/Context/ContextApi.php

```diff
@@ -33,7 +33,7 @@ class ContextApi extends BaseApiAbstract implements Waitable
      * @throws SmartlingApiException
      */
     public function wait(array $data) {
-        if (!empty($data['matchId'])) {
+        if (!empty($data['processUid'])) {
```

Comment:
`match` and `upload and match async` endpoint return `processUid` as well as old `matchId` (confirmed by Vitaly Barhatov). So, no backward compatibility issues here.